contracts/admin/src/state.rs

```diff
@@ -0,0 +1,82 @@
+use finance::currency::SymbolOwned;
```

Comment:
we do not want:
-  to mix the code of different entities, and
-  separate entity definition from its persistence

That is the `cw`-style we **do not follow**. Hence please:
- define each entity in its own module, and
- provide load/store functions in OOP style, as we have been doing for most of the other code